yarn.lock

```diff
@@ -0,0 +1,1852 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Does this project need an npm lock file and a yarn lock file? I think it's usually one tool or the other.

Reply:
Electron-builder strongly recommended using yarn to install instead of npm so I was afraid of any unforeseen bugs.  But I can change it if necessary.

Reply:
Using yarn is fine. I think you'll want to delete the npm package lock file then. Or do some research to see if that's necessary. I remember having both lock files seemed to cause problems with GardenGloss. Or it at least may cause some confusion for developers.

Reply:
Yes, I just read there would possibly be inconsistencies in the dependency tree when having both files.  Migrating was simple though since the yarn.lock file was already up-to-date.  I just deleted the pack-lock.json and pushed that commit.